packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -135,6 +135,12 @@ const EntityLayoutWrapper = (props: { children?: ReactNode }) => {
   );
 };
 
+/**
+ * TODO: For the MUI Grid to work there have to be "xs" set on every GridItem,
```

Comment:
Same. Items will equitably share the available space, if no breakpoints are set at all. If e.g. `md` or `lg` is set it will break. I updated the note to reflect on this :)